src/test/java/com/google/javascript/clutz/partial/missing_type.d.ts

```diff
@@ -0,0 +1,10 @@
+declare namespace ಠ_ಠ.clutz.module$exports$missing$type {
+  var x : any ;
```

Comment:
This should end up being `var x : Missing ;` eventually, but needs further changes, right?